janusgraph-backend-testutils/src/main/java/org/janusgraph/graphdb/JanusGraphIndexTest.java

```diff
@@ -3106,4 +3106,62 @@ public void shouldUpdateIndexFieldsAfterIndexModification() throws InterruptedEx
             graph2.close();
         }
     }
+    private void testMultipleOrClauses() {
```

Comment:
Can you please add ` clopen(option(FORCE_INDEX_USAGE), true);` to the beginning of this method, so that we are certain indexes are used.

Reply:
Strangely enough, if I add that line, the Solr tests fail while the ES tests are successful. I'm not familiar with Solr, any idea why the index is not being used in Solr's test case?

Reply:
similarly, adding the query to testGraphCentricQueryProfiling, the test will fail for Solr, but not for ES

Reply:
That's expected. You can use `if (indexFeatures.supportNotQueryNormalForm())` which evaluates to false for Solr.

Reply:
ah fair enough, updated the tests, thanks!